easycukes-scm/src/main/java/com/worldline/easycukes/scm/stepdefs/SCMStepdefs.java

```diff
@@ -202,5 +218,142 @@ public void commitInAnSVNRepository(String repository, String message)
                 DataInjector.injectData(message));
         log.info("Commit OK");
     }
+    
+    /**
+	 * Allows to create a branch in a given repository and execute git commit &
+	 * a git push command in the specified repository using the message provided
+	 * as a commit message
+	 * 
+	 * @param repository
+	 *            the location in which the git repository is located
+	 * @param message
+	 *            the message to be used as a commit message
+	 * @throws GitAPIException
+	 *             if something's going wrong while interacting with Git
+	 *             commands
+	 * @throws IOException
+	 *             if there's something wrong while committing the files
+	 */
+	@When("^I create a new branch in a git repository located in \"([^\"]*)\" having name \"([^\"]*)\" with the message \"([^\"]*)\"$")

```

Comment:
okk. Got your point will apply same for everywhere for branch, tag creation and deletion